blazar/plugins/oshosts/host_plugin.py

```diff
@@ -836,7 +836,8 @@ def notification_callback(self, event_type, payload):
                 if failed_hosts:
                     LOG.warn('%s failed.',
                              failed_hosts[0]['hypervisor_hostname'])
-                    reservation_flags = self._handle_failures(failed_hosts)
+                    for host in failed_hosts:
```

Comment:
Uhh... where is `_handle_failures` in our original fork? Was this just totally broken?? Or is it calling a superclass function

Reply:
Ouch. ouch ouch ouch